### PR TITLE
Use consistent project IDs between solutions

### PR DESCRIPTION
**Pull request by bricelam**
We don't want to run out of GUIDs... :trollface:

Just kidding, but this might help incremental builds between solutions (and build.cmd).

### CODE REVIEW

**Comment by anpete** [comment]
Anything need to change in OracleProvider.sln?

**Comment by smitpatel** [comment]
Minor thing but oracle provider should also be updated.

File: samples/OracleProvider/OracleProvider.sln [hunk 1]
```diff
@@ -3,34 +3,34 @@ Microsoft Visual Studio Solution File, Format Version 12.00
...
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "EFCore", "..\..\src\EFCore\EFCore.csproj", "{6934717E-F9CA-434F-B517-F20FCEFD3637}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "EFCore", "..\..\src\EFCore\EFCore.csproj", "{715C38E9-B2F5-4DB2-8025-0C6492DEBDD4}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "EFCore.Relational", "..\..\src\EFCore.Relational\EFCore.Relational.csproj", "{2FADFA29-0CFE-47A1-919A-00D243FD8F4F}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "EFCore.Relational", "..\..\src\EFCore.Relational\EFCore.Relational.csproj", "{6A25DF99-2615-46D8-9532-821764647EE1}"
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Framework", "Framework", "{57595541-E77D-477C-B48B-C69E061C11F9}"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "OracleProvider", "src\OracleProvider\OracleProvider.csproj", "{33FB559E-C3FC-4518-AC51-F92B53CCF580}"
```
Comment (smitpatel) [inline samples/OracleProvider/OracleProvider.sln]: This one is missing.

**Comment by smitpatel** [comment]
AppVeyor timed out. are you on latest dev rebase?

**Pull request merged**